src/client_side_reply.cc

```diff
@@ -1054,16 +1054,13 @@ clientReplyContext::purgeDoPurgeGet(StoreEntry *newEntry)
     assert (newEntry);
```

Comment:
Given that there was a isNull() check below and the DoPurgeHead() checks for nil, should this assert be replaced with a nil check and early return?
 or is it actually impossible for NullStoreEntry to occur here? (I lost track of the caller chain at 400 'created' methods)